tests/test-scenarios-github.sh

```diff
@@ -22,6 +22,13 @@ realpath() {
 TEST_HOME=`dirname $(realpath "$0")`
 CRD_HOME=`dirname $(realpath "$0")`/../deployments/CRD
 ARMOR_HOME=`dirname $(realpath "$0")`/../KubeArmor
+IGN_FILE=$TEST_HOME/tests.ignore
```

Comment:
we don't need the modification of test-scenarios-github.sh because we're not using cri-o in CI.
by the way, did you test ./test-scenarios-local.sh or ./test-scenarios-in-runtime.sh with multiubuntu_test_XX? if some test scenarios have CAP_NET_RAW, I think it is okay to add such scenarios in tests.ignore.

Reply:
I thought of adding these to tests.ignore but that would have ignored it for all container runtimes even the ones which are working fine right now, right? :thinking: